model/MailboxHandler.php

```diff
@@ -43,6 +43,11 @@ protected function initStruct() {
             # TODO: add virtual 'notified' column and allow to display who received a vacation response?
         );
 
+        if (Config::read('forgotten_user_password_reset')) {
+            $this->struct['phone'] = pacol(   1,          1,      0,      'text', 'pCreate_mailbox_phone'         , 'pCreate_mailbox_phone_desc'       , '');
+            $this->struct['email_other'] = pacol(   1,          1,      0,      'mail', 'pCreate_mailbox_email'         , 'pCreate_mailbox_email_desc'       , '');
+        }
+
```

Comment:
Much better, but the usual way to do this is to set display_in_list and display_in_form based on the config option (like $this->new is used for some fields). This way ensures the field gets filled with the correct default value if it's hidden from the user.